charts/newrelic-php-daemon/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for newrelic-php-daemon.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: newrelic/php-daemon
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "9.13.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
```

Comment:
Does this container really require a service account?

Reply:
done